our-brothers-web/src/app/app.component.scss

```diff
@@ -196,19 +196,180 @@
       }
     }
   }
+}
+@media only screen and (max-width: 1400px) {
+  .links-header {
+    .link-header {
+      &:not(:last-child) {
+        margin-left: 20px !important;
+      }
+    }
+  }
+  .logo-header {
+    background-repeat: no-repeat;
+    flex: 0 0 120px;
+    height: 50px;
+  }
+}
+
+@media only screen and (max-width: 700px) {
+  .header-out {
+    justify-content: center;
+  }
+  .header {
+    justify-content: center;
 
-  @media only screen and (max-width: 1400px) {
     .links-header {
-      .link-header {
-        &:not(:last-child) {
-          margin-left: 20px;
-        }
-      }
+      display: none;
     }
-    .logo-header {
-      background-repeat: no-repeat;
-      flex: 0 0 120px;
-      height: 50px;
+
+    .action-buttons {
+      display: none;
     }
   }
 }
+
+// Footer design
+.footer {
+  width: 100%;
+  position: relative;
+  height: auto;
+}
+.contain {
+  display: flex;
+  justify-content: center;
+}
+.footer .col {
+  width: 160px;
+  height: auto;
+  float: right;
+  box-sizing: border-box;
+  -webkit-box-sizing: border-box;
+  -moz-box-sizing: border-box;
+  padding: 0px 20px 20px 20px;
+}
+.footer .col-logo {
+  width: 275px;
+  padding-left: 65px;
+}
+.footer .col-double {
+  width: 105px;
+}
+.footer .col-last {
+  width: 225px;
+}
+.footer .col-copyright {
+  width: 80%;
+  text-align: center;
+  font-size: 12px;
+  color: #999999;
+}
+.footer .col h1 {
+  margin: 0;
+  padding: 0;
+  font-family: inherit;
+  font-size: 16px;
+  line-height: 17px;
+  padding: 20px 0px 5px 0px;
+  color: #00a099;
+  font-family: "arbel";
+}
+.footer .col ul {
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+}
+.footer .col ul li {
+  color: #999999;
+  font-size: 14px;
+  font-family: inherit;
+  // font-weight: bold;
+  line-height: 1.4em;
+  padding: 0px 0px 0px 0px;
+  cursor: pointer;
+  transition: 0.2s;
+  -webkit-transition: 0.2s;
+  -moz-transition: 0.2s;
+}
+.social ul li {
+  display: inline-block;
+  padding-right: 5px !important;
+}
+
+.footer .col ul li:hover {
+  color: #00a099;
+  transition: 0.1s;
+  -webkit-transition: 0.1s;
+  -moz-transition: 0.1s;
+}
+.clearfix {
+  clear: both;
+}
+.logo-footer {
+  margin-top: 35px;
+  height: 80px;
+  background-image: url(../assets/img/logo.png);
+  background-size: 100%;
+  background-repeat: no-repeat;
+}
+.site-hr {
+  max-width: 1050px;
+  margin-bottom: 20px;
+}
+.alignright {
+  float: right;
+  text-align: right;
+  width: 33.33333%;
+}
+.alignleft {
+  float: right;
+  text-align: left;
+  width: 33.33333%;
+}
+.aligncenter {
+  float: right;
```

Comment:
please use `flexbox` instead of `float`. See https://css-tricks.com/snippets/css/a-guide-to-flexbox/

Reply:
Fixed, I switched to flex

Reply:
So please remove all the `float` rules